p2p/p2p.go

```diff
@@ -188,3 +203,89 @@ func (s *Service) ListenAddrs() ([]multiaddr.Multiaddr, error) {
 
 	return listenAddrs, nil
 }
+
+// NewStream creates a bidirectional connection to a random peer that implements a set of protocol ids
+func (s *Service) NewStream(ctx context.Context, pids ...protocol.ID) (network.Stream, error) {
+	peers := s.host.Peerstore().Peers()
+	peersCount := peers.Len()
+	if peersCount < 0 {
```

Comment:
`peers.Len()` has this definition: `func (es IDSlice) Len() int           { return len(es) }`

what is the case when `len(...)` might return negative number?

Reply:
No idea, but builtin `len` returns an `int` as well

Reply:
I guess my question was why it's `peersCount < 0` and not `peersCount == 0`

Reply:
It should've been `<=`

Reply:
Fixed